db/seeds.rb

```diff
@@ -1,7 +1,18 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'open-uri'
+
+doc = Nokogiri::HTML(open("http://en365.ru/top500.htm"))
+
+cards = doc.css('td#middle>table tr:nth-child(n+2)').map do |el|
+  el.css('td').map(&:text)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.